model/postgres/postgres_server.rb

```diff
@@ -58,6 +58,14 @@ def configure_hash
 
     if resource.flavor == PostgresResource::Flavor::PARADEDB
       configs[:shared_preload_libraries] = "'pg_cron,pg_stat_statements,pg_analytics,pg_search'"
+    elsif resource.flavor == PostgresResource::Flavor::LANTERN
+      configs[:shared_preload_libraries] = "'pg_cron,pg_stat_statements,lantern_extras'"
+      configs[:"lantern.external_index_host"] = "'external-indexing.cloud.lantern.dev'"
+      configs[:"lantern.external_index_port"] = "443"
+      configs[:"lantern.external_index_secure"] = "true"
```

Comment:
Do we have any specific reason for keeping the keys as symbols? They'll be converted to strings when generating JSON anyway.

Reply:
No specific reason. I used symbols because existing configs were already symbols. I'll change them to strings.

Reply:
Just to confirm, we aren't accessing this config hash anywhere in the control plane, correct? It's solely for the data plane? If not, we'll need to update the locations where it's accessed as well.

Reply:
No, we don't access it in the control plane. We only need to update some tests.